Makefile

```diff
@@ -124,7 +123,17 @@ generate: gen-helm-docs ## Generate manifests, code, and docs resources
 ##@ Deployment
 
 .PHONY: deploy
-deploy: ## Deploy manager resources to the K8s cluster
+deploy: ## Deploy Reloader controller resources to the K8s cluster
+	kubectl create namespace bank-vaults-infra --dry-run=client -o yaml | kubectl apply -f -
+	$(HELM) upgrade --install vault-secrets-reloader deploy/charts/vault-secrets-reloader \
+		--set image.tag=dev \
+		--set collectorSyncPeriod=30s \
+		--set reloaderRunPeriod=1m \
+		--namespace bank-vaults-infra
+
+.PHONY: deploy-kind
+deploy-kind: ## Deploy Reloder controller resources to the kind cluster
+	kind load docker-image $(IMG) --name $(TEST_KIND_CLUSTER)
```

Comment:
Can we reuse `deploy` in `deploy-kind`? I'd suggest creating an additional target `upload-kind` (can be hidden), and using `deploy-kind: upload-kind deploy` as the final target.